src/lib/components/Journal.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from "react";
+
+import { GroupField, Input } from "react-invenio-forms";
+import { Grid } from "semantic-ui-react";
+
+import PropTypes from "prop-types";
+
+export class Journal extends Component {
```

Comment:
do you have any screenshots? I am curious how it looks like

Reply:
Here you go, sorry for the delay!

![Screenshot at Feb 27 15-25-30](https://user-images.githubusercontent.com/21204744/221589809-6f1eeaff-393c-423c-9246-665fee9ebddf.png)